integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CarbonShowCacheCommand.scala

```diff
@@ -205,4 +210,9 @@ case class CarbonShowCacheCommand(tableIdentifier: Option[TableIdentifier],
       }
     }
   }
+
+  def isValidTable(carbonTable: CarbonTable, sparkSession: SparkSession): Boolean = {
```

Comment:
dont need to redefine this function.

Reply:
`Checker.validateTableExists` throws an exception when the table is invalid, I don't need that in this case, I simply need that to be ignored.

Reply:
Got it.